app/controllers/partnership/BusinessUTRController.scala

```diff
@@ -14,6 +14,22 @@
  * limitations under the License.
  */
 
+/*
```

Comment:
Copyright notices are in twice

Reply:
strange, don't know how that happened. Anyway, fixed now